tests/parsenum/main.c

```diff
@@ -7,7 +7,8 @@
 #define PARSENUM_CMP(x, y)						\
 	((isnan((double)x) && isnan((double)y)) ||			\
 	    ((isinf((double)x) && isinf((double)y)) &&			\
-	     (((x > 0) && (y > 0)) || ((x < 0) && (y < 0)))) ||		\
+		((((double)x > 0) && ((double)y > 0)) ||		\
```

Comment:
FWIW, I'd write this as `(double)x * (double)y > 0` since we've already determined that they're both infinities...